src/ankh/helm/helm.go

```diff
@@ -249,3 +273,155 @@ func Template(ctx *ankh.ExecutionContext, ankhFile ankh.AnkhFile) (string, error
 
 	return finalOutput, nil
 }
+
+func getChartFileContent(ctx *ankh.ExecutionContext, path string, useContext bool, key string) ([]byte, error) {
```

Comment:
this seems like a good opportunity to make a new file `inspect.go`

Reply:
Done, added under the `helm` package - lmk if there's a more appropriate location